prefect_docker/host.py

```diff
@@ -39,6 +39,22 @@ class DockerHost(Block):
         max_pool_size: The maximum number of connections to save in the pool.
         client_kwargs: Additional keyword arguments to pass to
             `docker.from_env()` or `DockerClient`.
+
+    Examples:
+        Get a Docker Host client.
+        ```python
+        from prefect import flow
+        from prefect_docker import DockerHost
+
+        @flow
+        def docker_host_get_client_flow():
+            docker_host = DockerHost(
+                base_url="tcp://127.0.0.1:1234",
+                max_pool_size=4
+            )
+            with docker_host.get_client() as client:
+                pass
```

Comment:
Similar idea that the flow isn't strictly necessary
```suggestion
        from prefect_docker import DockerHost

        docker_host = DockerHost(
        base_url="tcp://127.0.0.1:1234",
            max_pool_size=4
        )
        with docker_host.get_client() as client:
            ... # Use the client for Docker operations
```